setup.py

```diff
@@ -55,6 +55,7 @@ def read(*paths):
               'plone.app.dexterity',
               'plone.app.testing',
               'mock',
+              'unittest2',
```

Comment:
@reinhardt: i'm curious why the unittest2 dependency is needed. could you please explain to me?